source/documentation/common/install/proc-Enabling_the_Red_Hat_Virtualization_Manager_Repositories.adoc

```diff
@@ -175,12 +175,19 @@ endif::ovirt-doc[]
 ----
 # dnf module -y enable nodejs:14
 ----
+ifdef::SHE_cli_deploy[]
+. Update the Self-Hosted Engine using the procedure  link:{URL_virt_product_docs}{URL_format}/upgrade_guide/index#Updating_a_self-hosted_engine_minor_updates[Updating a Self-Hosted Engine] in the _Upgrade Guide_.
```

Comment:
Is there an extra space after the word "procedure"? Other than that, LGTM -- no need for me to review this again.